contracts/SignerBondsUniswapV2.sol

```diff
@@ -241,6 +254,25 @@ contract SignerBondsUniswapV2 is ISignerBondsSwapStrategy, Ownable {
         emit UniswapV2SwapExecuted(amounts);
     }
 
+    /// @notice Approves the signer bonds swapper. The change takes effect
+    ///         immediately.
+    /// @dev Can be called only by the contract owner.
+    /// @param swapper Swapper that will be approved
+    function approveSwapper(address swapper) external onlyOwner {
+        emit SignerBondsSwapperApproved(swapper);
+        approvedSwappers[swapper] = true;
+    }
+
+    /// @notice Unapproves the signer bonds swapper. The change takes effect
+    ///         immediately.
+    /// @dev Can be called only by the contract owner.
+    /// @param swapper Swapper that will be unapproved
+    function unapproveSwapper(address swapper) external onlyOwner {
+        require(approvedSwappers[swapper], "Signer swapper is not approved");
```

Comment:
Why *signer swapper*? Should probably be *signer bonds swapper*

Reply:
Fixed in https://github.com/keep-network/coverage-pools/pull/148/commits/8b06db86517b95422ace7846320b14a1cbaee844